manifests/12_prometheus_rule.yaml

```diff
@@ -17,9 +17,8 @@ spec:
       rules:
         - alert: CommunityOperatorsCatalogError
           annotations:
-            summary: Community-operators not ready for 10 minutes
-            description: Fires whenever the community-operators source is not ready for more than 10 mins.
-            message: Default OperatorHub source "community-operators" is in Non-Ready state for more than 10 mins.
+            summary: Community-operators source is in non-ready for more than 10 minutes.
+            description: Operators shipped via this source are not available to install until the issue is fixed. Inspect the status of the pod owned by community-operators source in the openshift-marketplace namespace (oc get pods -n openshift-marketplace) to diagnose and repair. {{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} For more information refer to {{ label \"url\" (first $console_url ) }}/k8s/ns/openshift-marketplace/pods.{{ end }}{{ end }}" }}
```

Comment:
Worth additional impact clarification like "There is no effect on already installed operators"?  Or maybe "Installed operators dependent on this source will not receive updates until the issue is fixed"?  Or whatever the impact on installed operators actually is.

Reply:
Looks like you could use `oc -n openshift-marketplace get pods -l olm.catalogSource=community-operators` to trim the results down to just this pod, and similarly for the other alerts:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-serial/1514248134916575232/artifacts/e2e-aws-serial/gather-extra/artifacts/pods.json | jq -r '.items[].metadata | select(.namespace == "openshift-marketplace") | .name + " " + .labels["olm.catalogSource"]'
certified-operators-q6cdz certified-operators
community-operators-8lrw9 community-operators
marketplace-operator-7c5bf65dd6-wptp9 
redhat-marketplace-vs6rr redhat-marketplace
redhat-operators-nc492 redhat-operators
```